samples/migrations/milvus_migration_test.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import uuid
+from typing import Sequence
+
+import pytest
+import pytest_asyncio
+from migrate_milvus_vectorstore_to_alloydb import main
+from sqlalchemy import text
+from sqlalchemy.engine.row import RowMapping
+
+from langchain_google_alloydb_pg import AlloyDBEngine
+
+DEFAULT_TABLE = "test_milvus_migration" + str(uuid.uuid4())
```

Comment:
Can we use a milvus lite instance like mentioned in https://python.langchain.com/docs/integrations/vectorstores/milvus/